### ISSUE TITLE
UiTab - Custom title with HTML

### ISSUE THREAD

**Issue by Stormplay** [JosephusPaye/Keen-UI#247]
Please add сustom title with HTML.

**Comment by JosephusPaye** [comment]
Hi @Stormplay,

Thanks for the suggestion. I'll see how it can be added.

**Comment by JosephusPaye** [comment]
Added in `next`, to be released soon. Use the `header` slot for custom HTML.

```vue
<ui-tabs>
    <ui-tab>
        <div slot="header">Tab 1: <b>HTML</b></div>
        <p>A link to <a href="https://github.com">Github</a>.</p>
    </ui-tab>
    <ui-tab>
        <div slot="header">Tab 2: <i>HTML</i></div>
        <p>A link to <a href="https://github.com">Github</a>.</p>
    </ui-tab>
</ui-tabs>
```

**Comment by JosephusPaye** [comment]
Released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
